src/systems/ec/implementations/vanilla/packages/ec-component-date-block/ec-component-date-block.scss

```diff
@@ -6,119 +6,163 @@
 // Import base
 @import '@ecl/ec-base/ec-base';
 
-@mixin ecl-date-block() {
+@mixin ecl-date-block(
+  $background-default: $ecl-color-blue,
+  $text-color-default: $ecl-color-white,
+  $separator-color-default: $ecl-color-grey-25,
+  $year-background-default: $ecl-color-grey-10,
+  $year-text-color-default: $ecl-color-grey,
+  $background-ongoing: $ecl-color-yellow,
+  $text-color-ongoing: $ecl-color-grey,
+  $separator-color-ongoing: $ecl-color-grey-25,
+  $year-background-ongoing: $ecl-color-grey-10,
+  $year-text-color-ongoing: $ecl-color-grey,
+  $background-canceled: $ecl-color-grey-10,
+  $text-color-canceled: $ecl-color-grey,
+  $separator-color-canceled: $ecl-color-grey-25,
+  $year-background-canceled: $ecl-color-grey-10,
+  $year-text-color-canceled: $ecl-color-grey,
+  $background-past: $ecl-color-white,
+  $text-color-past: $ecl-color-grey,
+  $separator-color-past: $ecl-color-grey-50,
+  $year-background-past: $ecl-color-white,
+  $year-text-color-past: $ecl-color-grey,
+  $width-mobile: 5rem,
+  $width-desktop: 6.3rem,
+  $circle-width: 0.6rem
+) {
   .ecl-date-block {
-    align-items: center;
-    background-color: map-get($ecl-colors, 'grey-5');
-    border-bottom: 4px solid map-get($ecl-colors, 'yellow-100');
-    box-sizing: border-box;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
+    background-color: $background-default;
+    display: block;
```

Comment:
`box-sizing: border-box;` is missing

Reply:
there is no display difference when using the fake global rules, so why should we add this line?

Reply:
there is a difference ("past" variant)

Reply:
indeed